main.go

```diff
@@ -110,25 +230,60 @@ func main() {
 	filePath := os.Args[1]
 	debug("## Loading " + filePath)
 
-	fileData, err := ioutil.ReadFile(filePath)
+	var subsetPartition map[int][]int
+	err, test := readTestFile(filePath)
```

Comment:
`test, err :=`.  I have never seen a case where err is the first thing returned by a function